templates/helm/command-center/templates/argo.yaml

```diff
@@ -0,0 +1,72 @@
+---
+{{- $secretName := "db-password" }}
+{{- $keyName := "password" }}
+{{- $env := .Values.env }}
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: db-login-secrets
+spec:
+  releaseName: db-login-secrets
+  targetNamespace: secrets-manager
+  resetValues: true
+  chart:
+    repository: https://broadinstitute.github.io/datarepo-helm
+    name: create-secret-manager-secret
+    version: 0.0.5
+  values:
+    secrets:
+      {{- range (append .Values.argoNamespaces "argo-ui") }}
+      - secretName: {{ $secretName }}
+        nameSpace: {{.}}
+        vals:
+          - kubeSecretKey: {{ $keyName }}
+            path: secret/dsde/monster/{{ $env }}/command-center/cloudsql/users/argo
+            vaultKey: password
+      {{- end }}
+{{- range .Values.argoNamespaces }}
```

Comment:
this is nifty! The syntax here took me a little bit to understand, and since this `range` statement is before an entire yaml section for a controller per namespace, I think it miiight be helpful to throw in a quick little comment before it; the "---" line right after this confused me for a bit and I thought it was a typo. Thoughts?